HISTORY.md

```diff
@@ -1,59 +1,63 @@
-##0.16.5
-**Changes**
+# ripple-lib Release History
+
+## 0.16.5 (2016-01-21)
```

Comment:
I notice you only added the date here. Is the idea of include the date on the latest version or to include it on all releases going forward?

Reply:
All releases going forward. I didn't go back to add it to older releases because I didn't think it would be worth the effort :)